### DISCUSSION TITLE
publish MQTT when value changed more then 100

### DISCUSSION THREAD

**Discussion by bananajoe86** [arendst/Tasmota#13018] [Q&A] [answered]
Hey community,

I need your help. I've a HX711 Sensor. At the moment, I have a teleperiod 10 and recieve via MQTT every 10 Seconds the witghtraw. 
My target is, not sending every 10 Seconds a MQTT, but only when the Sensor change the value more then 100grams.
Neither with sensor34 9 or a rule I reach my goal.
Mayby someone has an idea for my.
Thx Joe

**Comment by ascillato** [comment, marked as answer]
https://tasmota.github.io/docs/Rules/#transmit-sensor-value-only-when-a-delta-is-reached
